Cargo.toml

```diff
@@ -50,12 +53,15 @@ pathdiff = { version = "0.2.1", features = ["camino"] }
 minijinja = { version = "1.0.3", features = ["loader", "preserve_order"] }
 include_dir = "0.7.3"
 slug = "0.1.4"
-directories = "5.0.1"
+oranda-generate-css = { path = "oranda-css" }
```

Comment:
any reason to have the crate not have the same name as the dir..?

Reply:
Also you need to have an actual version here or `cargo publish` will fail, as published crates can only depend on crates in crates.io. In general making this a crate means we now need to publish this to crates.io as its own thing.

The PublishRelease ci task will also need to be updated to `cargo publish ...` this package before publishing oranda.

Also versioning this in lockstep with oranda itself will streamline release processes.

(I possibly led you astray with the note of ".crates are just a tarball of the whole dir" because I'm *pretty sure* that the obvious implications of that re: path dependencies doesn't work -- cargo/crates.io will still complain on publish.)

Reply:
> any reason to have the crate not have the same name as the dir..?

mainly because the directory also contains the actual oranda css, and it feels like signalling "oh this only contains rust code" if we name it that way. but we could also separate the directories properly

Reply:
good call on the version, that's kind of annoying but makes sense from Cargo's perspective

Reply:
okay, `oranda-generate-css` is published, i'll set the correct perms tomorrow